app/views/likes/new.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Like this comment</h1>
+<%= form_with model:@comment, url: user_post_likes_path, method: :post,local: true do |form| %>
+    <%= form.submit 'Like' %>
+<% end %>
```

Comment:
- **[OPTIONAL]** There is no need for a new like page. You should directly `create` the like when a user clicks the like button instead of taking them to a new page.